docs/sharing/design-integrations.md

```diff
@@ -0,0 +1,101 @@
+
+![Storybook addon figma](./storybook-figma-addon.png)
+
+Or via a plugin developed by the Storybook maintainers. [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) is the official Figma plugin that lets you link stories to your Figma components. You can preview your existing stories in the Figma design workspace when configured. Below is an overview of how to get started with the plugin.
+
+### Setup
+
+To use the plugin, you must have a Storybook [published in Chromatic](./publish-storybook.md#publish-storybook-with-chromatic), a free publishing service made for Storybook that documents, versions, and indexes your UI components securely in the cloud. You'll be using the URL of your published Storybook to connect to the plugin.
+
+Navigate to [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) to install the plugin. In Figma, open the command palette (in Mac OS, use `Command + /`, in Windows use `Control + /`) and type `Storybook Connect` to enable it.
+
+![Figma palette Storybook connect](./figma-plugin-open-in-figma.png)
+
+Follow the instructions to connect and authenticate with Chromatic.
+
+### Link stories to Figma components
+
+The core feature of the [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) Figma plugin is the ability to connect your existing components, variants, and instances to Storybook stories.
+
+To link your components with Storybook stories, navigate to the story published on Chromatic, ensure that you're using the correct branch, and copy the link.
+
+In Figma, open the plugin and paste the URL.
+
+![Story linked in Figma](./figma-plugin-paste-url.png)
+
+Once they're connected, you'll be able to preview the story by clicking the link in the sidebar.
+
+<div class="aside">
+💡 The plugin does not support linking stories to Figma layers.
+</div>
+
+### Preview stories in Figma
+
+Another essential feature of the plugin is the ability to preview the story directly in your Figma workspace. To preview the story connected to your existing component, open Figma's design sidebar and click the "View story" action.
+
+<video autoPlay muted playsInline loop>
+  <source src="figma-plugin-open-story.mp4" type="video/mp4" />
+</video>
+
+## Adobe XD
+
+[Adobe XD](https://www.adobe.com/products/xd.html) is a vector-based UI and UX design tool, enabling design teams to create wireframes, interactive designs, prototypes, or hi-fidelity web or application designs.
+
+Adobe XD integrates with Storybook via [addon](https://storybook.js.org/addons/storybook-addon-xd-designs), enabling you to embed your design specs alongside your Storybook stories.
```

Comment:
This addon is no longer maintained. I was able to get it working using the iframe option of the designs addon: https://pocka.github.io/storybook-addon-designs/?path=/story/docs-iframe-readme--page

Reply:
@winkerVSbecks thanks for the callout on this! Appreciate it :)! I've updated the section accordingly.